### PR TITLE
Feature busker model

### PR DESCRIPTION
**Pull request by diogo-botelho**
- Created busker model, routes, sql schema and json schema.
- Created BuskerData interface.
- Modified helpers/sql to include BuskerData.

### CODE REVIEW

File: backend/src/helpers/sql.ts [hunk 1]
```diff
@@ -23,12 +26,12 @@ interface EventData {
...
-  lastName?: string;
-  phone?: string;
-  email?: string;
-}
+// interface UserData {
+//   firstName?: string;
+//   lastName?: string;
+//   phone?: string;
+//   email?: string;
+// }
```
Comment (samau3) [inline backend/src/helpers/sql.ts]: Can these commented out lines be removed?

File: backend/src/models/busker.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+   * all the fields; this only changes provided ones.
+   *
+   * Data can include:
+   *   { buskerName, category, description }
+   *
+   * Returns { id, buskerName, category, description }
+   *
+   * Throws NotFoundError if not found.
+   *
+   * WARNING: this function can set a new password or make a user an admin.
```
Comment (samau3) [inline backend/src/models/busker.ts]: Should updating a busker to be an admin be possible via the API, or should that be restricted to only be changed via the database?

**Review by samau3 — COMMENTED** [review]
Left some comments, but none are blocking so feel free to push forward if needed!

File: backend/src/helpers/sql.ts (hunk 1 above)
Comment (diogo-botelho) [reply backend/src/helpers/sql.ts]: Done

File: backend/src/models/busker.ts (hunk 2 above)
Comment (diogo-botelho) [reply backend/src/models/busker.ts]: Solved - was an outdated docstring.

**Review by samau3 — APPROVED** [review]

**Pull request merged**